tests/test_start_sim.py

```diff
@@ -30,5 +28,5 @@ def on_reset(self):
     args = parser.parse_args()
 
     env = Sim(args.aircraftconfig, args.gymfc_config, args.verbose)
-    env.render()
+    env.render().wait()
```

Comment:
Ah this will block until you close the Gazebo UI correct?

Reply:
Exactly. When running this with Ubuntu this does not matter as the gazebo client process stays running even after the python script ends. But the docker image would terminate together with the main process if it does not block.

Reply:
Ah ok that's probably why I never noticed an issue.